web/packages/plugins/ui/tabs/src/TabsContext.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { createContext, useContext } from 'react'
+
+// @ts-ignore
+export const TabsContext = createContext<any>({
+  fields: {},
+  widgets: {},
+})
```

Comment:
Did we not move away from having Tabs as a Provider?
Also why do Tabs care or need to know about fields and widgets?

Reply:
I think we can keep the provider for tabs, so we don't need to pass down the onOpen through all the nested children. I will remove the fields and widgets, copy paste error.